tests/test_cleedpy_yaml_adapter.py

```
@@ -0,0 +1,39 @@
+import pytest
+
+
+yaml = pytest.importorskip("yaml")
+
+from phaseshifts.io import (  # noqa: E402
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)